libs/damap/src/lib/services/form.service.ts

```diff
@@ -484,11 +484,22 @@ export class FormService {
   }
 
   private createCostFormGroup(): UntypedFormGroup {
+    let costFormControl = new FormControl(0, {
+      validators: currencyValidator(),
+      updateOn: 'change',
+    });
+    costFormControl.valueChanges.subscribe(n => {
```

Comment:
Try avoid using `.subscribe()` as `Subscriptions` exist until they are unsubscribed (unless it's a HTTP call). As this function will get called multiple times, we'll end up with mutliple (unused) subscriptions (see also: https://rxjs.dev/guide/observable#disposing-observable-executions).

Reply:
Good point, thank you :)
In general, I am not sure if this is the right place to perform this conversion. It would be nice to have it here, so that further access does not have to make sure it is a number. On the other hand, it is not really used anywhere else. Which means, it would suffice to perform this conversion when sending the DMP to the back-end. Which will only happen, if the form is valid - thus there will always be a valid number in the text field (with or without decimals). What do you think?